### PR TITLE
Eduardo Santana Homework #1 Resubmit

### PR DESCRIPTION
**Pull request by Ed-San1**
The two previous commits I had made on the readme were lost but essentially they involved some minor formatting changes because I realized I had not properly spaced the headers from the text and had not properly linked an image. I was already familiar with the syntax for markdown from my time using the website, Reddit, so this assignment was not overly difficult to prepare! 

This was originally done on RStudio Cloud and that presented no issues with regards to the workflow of homework assignment.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,20 @@
...
\ No newline at end of file
+# Homework 1 
+
+![Me](https://tinyurl.com/y5ybnl62)
+
+## About Me
+
+### Blurb about Home
+I am originally from Miami, FL but, currently, my family lives in ~~south~~ southwest Florida. People who are from Miami proper insist that you refer to my part as **SOUTHWEST** Florida, rather than as *South* Florida because they somehow have sole ownership of the direction "south".
+
```
Comment (Tracylyy) [inline README.md:9]: It's really good to use headers!!

Comment (Tracylyy) [reply README.md:9]: Headers always help people to understand what you are doing. Also, good use of bold.

File: README.md [hunk 2]
```diff
@@ -1 +1,20 @@
...
+### Languages
+I spreak three languages with varying levels of fluency:
+1. English (Native)
+2. Español/Spanish (Native)
+3. Русский/Russian (Advanced)
+
```
Comment (Tracylyy) [inline README.md:15]: I really like how you used the ordered list.

File: README.md [hunk 3]
```diff
@@ -1 +1,20 @@
-# hw01
\ No newline at end of file
+# Homework 1 
+
+![Me](https://tinyurl.com/y5ybnl62)
+
+## About Me
+
+### Blurb about Home
+I am originally from Miami, FL but, currently, my family lives in ~~south~~ southwest Florida. People who are from Miami proper insist that you refer to my part as **SOUTHWEST** Florida, rather than as *South* Florida because they somehow have sole ownership of the direction "south".
```
Comment (Adam-Fofana) [inline README.md:8]: Disputes over region names are always fun.

File: README.md [hunk 4]
```diff
@@ -1 +1,20 @@
...
+
+### Languages
+I spreak three languages with varying levels of fluency:
+1. English (Native)
+2. Español/Spanish (Native)
+3. Русский/Russian (Advanced)
+
+I studied Russian throughout undergrad and found it to be very fulfilling, in no small part thanks to its very rich literarure and folklore! Below are some of the more interesting Russian folksongs I have encountered!
+
+- [The Cossack Lullaby](https://youtu.be/cJsTB-yB-uk)
```
Comment (Adam-Fofana) [inline README.md:18]: Good use of links.

**Review by Adam-Fofana — COMMENTED** [review]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |     x      |              |            |
| **Coding strategy**         |    x       |              |            |
| **Presentation**            |        x   |              |            |
| **Achievement, creativity** |   x        |              |            |
| **Ease of access**          |       x    |              |            |

File: README.md (hunk 3 above)
Comment (bensoltoff) [inline README.md:8]: It's good practice to leave a blank line between headers and blocks of text. It avoids formatting issues when the document is rendered

File: README.md [hunk 5]
```diff
@@ -1 +1,20 @@
...
+![Me](https://tinyurl.com/y5ybnl62)
+
+## About Me
+
+### Blurb about Home
+I am originally from Miami, FL but, currently, my family lives in ~~south~~ southwest Florida. People who are from Miami proper insist that you refer to my part as **SOUTHWEST** Florida, rather than as *South* Florida because they somehow have sole ownership of the direction "south".
+
+### Languages
+I spreak three languages with varying levels of fluency:
+1. English (Native)
```
Comment (bensoltoff) [inline README.md:12]: You actually don't have to number the elements in the list. Markdown does that automatically.

```
1. English (Native)
1. Español/Spanish (Native)
1. Русский/Russian (Advanced)
```

1. English (Native)
1. Español/Spanish (Native)
1. Русский/Russian (Advanced)

**Review by bensoltoff — COMMENTED** [review]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |           |       x       |            |
| **Coding strategy**         |           |    x          |            |
| **Presentation**            |     x      |              |            |
| **Achievement, creativity** |      x     |              |            |
| **Ease of access**          |     x      |              |            |

**Pull request closed without merging**